### DISCUSSION TITLE
Rename Fn / Win swap to Fn / Super swap

### DISCUSSION THREAD

**Discussion by teackot** [BeardOverflow/msi-ec#21] [General]
When I sent the first patch to LKML, I renamed Win to Super as this key is usually called Super or Meta in Linux. Should we rename it to Super here too, or maybe rename it back to Win in the kernel?

**Comment by BeardOverflow** [comment]
It's hard to answer.

I did a fast search in google and I found that:

- Windows calls it Windows key
- Apple calls it Command key
- Gnome calls it Super key
- KDE calls it Meta key

However, digging in the kernel code, I found this: https://github.com/torvalds/linux/blob/master/include/uapi/linux/input-event-codes.h#L203-204

Searching for KEY_LEFTMETA and KEY_RIGHTMETA, I was able to find a lot of entries between commits and source code.

So, following the kernel code, it seems that the correct answer is to call it Meta key.

**Comment by teackot** [comment]
We had a discussion in the LKML earlier and decided that keeping the "Fn / Win swap" name is the way to go, because at this point renaming it to Super in this repo would be pretty tedious with all those open PRs. The upstream version of the driver doesn't use fn_[win/super]_swap config yet, so it was easy to rename.
